crates/flux-middle/src/rty/mod.rs

```diff
@@ -233,6 +233,7 @@ pub struct AssocPredicate {
     /// [`DefId`] of the container, i.e., the impl block or trait.
     pub container_def_id: DefId,
     pub name: Symbol,
+    pub span: Span,
```

Comment:
We should avoid having spans at `rty` because types could come for external crates for which we may not have spans.

Reply:
removed.